DataConnectors/Citrix_ADC_WAF.json

```diff
@@ -0,0 +1,116 @@
+{
+    "id": "Citrix",
+    "title": "Citrix ADC WAF",
+    "publisher": "Citrix System",
+    "descriptionMarkdown": "Common Event Format (CEF) is an industry standard format on top of Syslog messages, used by many security vendors to allow event interoperability among different platforms. By connecting your CEF logs to Azure Sentinel, you can take advantage of search & correlation, alerting, and threat intelligence enrichment for each log.",
```

Comment:
This description is just the generic CEF connector description. Please rewrite to customize to Citrix Netscaler ADC WAF and would be great if you can also expand on what ADC and WAF stands for as well so that customers get the complete picture. It helps to expand the acronym in one place at least.

Reply:
Done. Added Citrix WAF related content here.